widgets/PeopleActionButtonsMail.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace humhub\modules\mail\widgets;
+
+use humhub\modules\mail\Module;
+use humhub\modules\ui\icon\widgets\Icon;
+use humhub\modules\user\widgets\PeopleActionButtons;
+use humhub\widgets\BootstrapComponent;
+use Yii;
+
+class PeopleActionButtonsMail extends PeopleActionButtons
+{
+    public function run()
+    {
+        $html = $this->addFollowButton();
+        $html .= $this->addFriendshipButton();
```

Comment:
I would suggest that we replace this button/method with "message" if you are friends. What do you think about that? 

3 buttons are too much and probably look too much or? 
Can you post a screenshot of it?

Reply:
> I would suggest that we replace this button/method with "message" if you are friends. What do you think about that? 
> 
> 3 buttons are too much and probably look too much or? 
> Can you post a screenshot of it? 

An alternative would be using icons over basic text based buttons, better control over the size.